administration/src/App.tsx

```diff
@@ -51,19 +51,20 @@ const App = () => <AuthProvider>
                         <HashRouter>
                             <Navigation onSignOut={onSignOut}/>
                             <Main>
-                                <Route exact path={"/"}>
-                                    <div style={{display: 'flex', alignItems: 'center', flexDirection: 'column' }}>
+                                <Routes>
+                                    <Route path={"/"} element={<div
+                                        style={{display: 'flex', alignItems: 'center', flexDirection: 'column'}}>
                                         <H3>Wählen Sie eine Aktion aus:</H3>
-                                        <NavLink to={"/applications"}><Button style={{ marginBottom: '10px'}} icon="form" text="Eingehende Anträge"/></NavLink>
-                                        <NavLink to={"/eak-generation"}><Button icon="people" text="Karten erstellen"/></NavLink>
-                                    </div>
-                                </Route>
-                                <Route exact path={"/applications"}>
-                                    <ApplicationsController token={authData.token}/>
-                                </Route>
-                                <Route path={"/eak-generation"}>
-                                    <GenerationController/>
-                                </Route>
+                                        <NavLink to={"/applications"}><Button style={{marginBottom: '10px'}}
+                                                                              icon="form"
+                                                                              text="Eingehende Anträge"/></NavLink>
+                                        <NavLink to={"/eak-generation"}><Button icon="people"
+                                                                                text="Karten erstellen"/></NavLink>
+                                    </div>}/>
+                                    <Route path={"/applications"}
+                                           element={<ApplicationsController token={authData.token}/>}/>
+                                    <Route path={"/eak-generation"} element={<GenerationController/>}/>
+                                </Routes>
                             </Main>
```

Comment:
Why did you remove the `exact` flags for Route `/applications` and `/`?

Reply:
its no longer supported, should still work, at least from my tests